src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -23816,27 +23861,32 @@ interface I14<T> : I1<T> where T : I1<T>
 
         [Theory]
         [CombinatorialData]
-        public void ImplementAbstractStaticConversionOperator_04([CombinatorialValues("implicit", "explicit")] string op, bool structure)
+        public void ImplementAbstractStaticConversionOperator_04([CombinatorialValues("implicit", "explicit")] string op, bool structure, bool isChecked)
         {
+            if (GetConversionOperatorName(op, isChecked, out string checkedKeyword) is null)
+            {
+                return;
```

Comment:
Same comment for the rest of these.

Reply:
> Same comment for the rest of these.

Same response. I see no point in adding any asserts here. This is a part of test infrastructure rather than a validation path.